src/components/AgendaItemCard.tsx

```diff
@@ -282,3 +294,271 @@ export function SearchResultAgendaItemCard({
     </Link>
   );
 }
+
+type Motion = {
+  committeeSlug: string;
+  motionType: string;
+  motionId: string;
+  voteDescription: string;
+  dateTime: string;
+  value: string;
+  tally: string;
+  resultKind: string;
+  committeeName: string;
+};
+
+type SearchPageAgendaItemCardProps = {
+  item: AgendaItem;
+  decisionBody: DecisionBody;
+  categories: string[];
+  motions: Motion[];
+};
+
+export function SearchPageAgendaItemCard({
```

Comment:
Might make sense to put this new stuff in beside the app/search/page since we don't expect other people to consume this component